[2024-05-29T19:55:20.752+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.is_api_available manual__2024-05-29T19:55:08.791318+00:00 [queued]>
[2024-05-29T19:55:20.778+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.is_api_available manual__2024-05-29T19:55:08.791318+00:00 [queued]>
[2024-05-29T19:55:20.780+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T19:55:20.781+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-05-29T19:55:20.782+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T19:55:20.827+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-05-29 19:55:08.791318+00:00
[2024-05-29T19:55:20.842+0000] {standard_task_runner.py:55} INFO - Started process 6506 to run task
[2024-05-29T19:55:20.856+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '1_dag', 'is_api_available', 'manual__2024-05-29T19:55:08.791318+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/1_dag.py', '--cfg-path', '/tmp/tmpi875qnl3']
[2024-05-29T19:55:20.862+0000] {standard_task_runner.py:83} INFO - Job 48: Subtask is_api_available
[2024-05-29T19:55:21.137+0000] {task_command.py:376} INFO - Running <TaskInstance: 1_dag.is_api_available manual__2024-05-29T19:55:08.791318+00:00 [running]> on host 4b56679d13b8
[2024-05-29T19:55:22.437+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=1_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-05-29T19:55:08.791318+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-29T19:55:08.791318+00:00
[2024-05-29T19:55:22.452+0000] {http.py:120} INFO - Poking: api/
[2024-05-29T19:55:22.506+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-05-29T19:55:22.522+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-05-29T19:55:23.015+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-05-29T19:55:23.057+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=1_dag, task_id=is_api_available, execution_date=20240529T195508, start_date=20240529T195520, end_date=20240529T195523
[2024-05-29T19:55:23.258+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-05-29T19:55:24.138+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
